content/ko/docs/concepts/workloads/controllers/job.md

```diff
@@ -255,34 +286,41 @@ _작업 큐_ 잡은 `.spec.completions` 를 설정하지 않은 상태로 두고
 `.spec.completionMode` 에 지정된 완료 모드를 가질 수 있다.
 
 - `NonIndexed` (기본값): `.spec.completions` 가 성공적으로
-  완료된 파드가 있는 경우 작업이 완료된 것으로 간주된다. 즉, 각 파드
-  완료는 서로 상동하다(homologous). null `.spec.completions` 가 있는
-  잡은 암시적으로 `NonIndexed` 이다.
+  완료된 파드가 있는 경우 잡이 완료된 것으로 간주된다. 즉, 각 파드
+  완료는 서로 상동하다(homologous). `.spec.completions` 가 null인 
+  잡은 암시적으로 `NonIndexed` 상태가 된다.
 - `Indexed`: 잡의 파드는 연결된 완료 인덱스를 0에서 `.spec.completions-1` 까지 
-  가져온다. 이 인덱스는 다음의 세 가지 메카니즘으로 얻을 수 있다.
+  가져온다. 이 인덱스는 다음의 네 가지 메카니즘으로 얻을 수 있다.
   - 파드 어노테이션 `batch.kubernetes.io/job-completion-index`.
-  - 파드 호스트네임 중 일부(`$(job-name)-$(index)` 형태).
+  - Pod 레이블 `batch.kubernetes.io/job-completion-index` (v1.28 이상). 참고
+    이 레이블을 사용하려면 기능 게이트 `PodIndexLabel`을 활성화해서 하며 
+    기본적으로 활성화되어 있다. 
+  - 파드 호스트네임의 일부로 `$(job-name)-$(index)` 형태를 따른다.
     인덱스된(Indexed) 잡과 
     {{< glossary_tooltip text="서비스" term_id="Service" >}}를 결합하여 사용하고 있다면, 잡에 속한 파드는
     DNS를 이용하여 서로를 디스커버 하기 위해 사전에 결정된 호스트네임을 사용할 수 있다.
-    이를 어떻게 설정하는지에 대해 궁금하다면, [파드 간 통신을 위한 잡](/ko/docs/tasks/job/job-with-pod-to-pod-communication/)를 확인한다.
+    이를 어떻게 설정하는지에 대해 궁금하다면, [파드 간 통신을 위한 잡](/docs/tasks/job/job-with-pod-to-pod-communication/)을 확인한다.
   - 컨테이너화된 태스크의 경우, 환경 변수 `JOB_COMPLETION_INDEX`에 있다.
 
   각 인덱스에 대해 성공적으로 완료된 파드가 하나 있으면 작업이 완료된 것으로
   간주된다. 이 모드를 사용하는 방법에 대한 자세한 내용은
-  [정적 작업 할당을 사용한 병렬 처리를 위해 인덱싱된 잡](/ko/docs/tasks/job/indexed-parallel-processing-static/)을 참고한다.
-  참고로, 드물기는 하지만, 동일한 인덱스에 대해 둘 이상의 파드를 시작할 수
-  있지만, 그 중 하나만 완료 횟수에 포함된다.
+  [정적 작업 할당을 사용한 병렬 처리를 위해 인덱싱된 잡](/docs/tasks/job/indexed-parallel-processing-static/)을 참고한다.
 
+{{< note >}}
+드물지만, 다양한 이유(노드 장애, kubelet 재시작 또는 파드 제거 등)로 
+동일한 인덱스에 대해 여러 파드가 시작될 수 있다. 이 경우, 성공적으로 완료된 첫 번째 파드만 
+완료 ​​횟수에 포함되고 잡 상태가 업데이트된다. 동일한 인덱스에 대해 실행 중이거나 완료된 다른 파드는 
+감지되는 즉시 잡 컨트롤러에 의해 삭제된다.
+{{< /note >}}
 
 ## 파드와 컨테이너 장애 처리하기
 
-파드내 컨테이너의 프로세스가 0이 아닌 종료 코드로 종료되었거나 컨테이너 메모리 제한을
-초과해서 죽는 등의 여러가지 이유로 실패할 수 있다.  만약 이런 일이
+파드 내 컨테이너는 프로세스가 0이 아닌 종료 코드로 종료되었거나 컨테이너 메모리 제한을
+초과해서 죽는 등의 여러가지 이유로 실패할 수 있다.  만약 이런 상황이 
 발생하고 `.spec.template.spec.restartPolicy = "OnFailure"` 라면 파드는
 노드에 그대로 유지되지만, 컨테이너는 다시 실행된다.  따라서 프로그램은 로컬에서 재시작될 때의
 케이스를 다루거나 `.spec.template.spec.restartPolicy = "Never"` 로 지정해야 한다.
-더 자세한 정보는 [파드 라이프사이클](/ko/docs/concepts/workloads/pods/pod-lifecycle/#상태-예제)의 `restartPolicy` 를 본다.
+더 자세한 정보는 [파드 라이프사이클](/docs/concepts/workloads/pods/pod-lifecycle/#example-states)의 `restartPolicy` 를 본다.
```

Comment:
There is no #example-states on the doc. I have finally found a root cause and a PR to English doc has been raised by myself: https://github.com/kubernetes/website/pull/53818